DelegationStationTests/Pages/TagEditTest.cs

```diff
@@ -389,7 +389,8 @@ public void SaveButtonShouldRender()
                 var cut = RenderComponent<TagEdit>(parameters => parameters
                     .Add(p => p.Id, "myId"));
                 var buttonElement = cut.Find("#SaveButton");
-                buttonElement.Click();
+                //FIXME:  Do we actually need this to test?
```

Comment:
I don't think so but we do want to add a test method to validate that the save button runs the method with the expected results